foundation/foundation-mda/src/main/resources/templates/deployment/vault/vault.values.yaml.vm

```diff
@@ -6,11 +6,10 @@ replicaCount: 1
 
 hostname: vault
 image:
-  name: ${projectName}-vault-docker
-  imagePullPolicy: Always
-  dockerRepo: ${dockerProjectRepositoryUrl}
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  name: boozallen/aissemble-vault
+  imagePullPolicy: IfNotPresent
+  dockerRepo: ghcr.io/
+  tag: "${versionTag}"
```

Comment:
I: Update vault deployment to use the aiSSEMBLE image by default rather than a docker image within the project. User would have to create a custom execution for adding the vault docker image build to their project. The dockerfile that is then added, doesn't enable any additional functionality (see the [dockerfile template](https://github.com/boozallen/aissemble/blob/dev/foundation/foundation-mda/src/main/resources/templates/general-docker/vault.docker.file.vm)). Makes it easier for the user to use the vault deployment out of the box.

Reply:
Deprecated